datalake/src/main/resources/duties/7.2.15/aws/cdp_data_lake_medium_duty_with_profiler/medium_duty_ha.json

```diff
@@ -49,8 +49,8 @@
         "attachedVolumes": [
           {
             "count": 1,
-            "size": 250,
```

Comment:
Why is this 1TB instead of the 512 we agreed on for other shapes?

Reply:
fixed